### ISSUE TITLE
Identify main sections needed in paper

### ISSUE THREAD

**Issue by ZacheryPD** [NCSU-CSC510-Group-E/WolfTutor#20]

**Comment by mgmetro4** [comment]
What do you guys think?
I can rework the architecture a bit and I think our introduction is pretty good. and our enhancement section? Except maybe we could replace chosen enhancement stuff with what actually went down (not just abstract thinking)

Not sure what else we should add aside from evaluations and future scope, which TA recommended.

**Comment by zhazhangqi** [comment]
How about talks some of our simulated data? How we generated , what kind of data we gernerated for which scenario .

**Comment by mgmetro4** [comment]
It looks like the second projects from 2017 and 2016 were about bad smells and github activity analysis based on the first round projects. tried to find papers for our classmates, but the githubs I looked at had nothing. I feel like We can just take our proposal though and add to Evaluations, Future Scope, and the timeline stuff about what we actually did.

@zhazhangqi Yea, we should probably have a Dataset subsection under evaluations for that. I need to look into normal distributions still to see if that means anything for us. Generating graphs in excel would only take me a second though
